dora/stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java

```diff
@@ -402,6 +390,10 @@ private final class BenchThread implements Callable<Void> {
     private final byte[] mBuffer;
     private final WorkerBenchTaskResult mResult;
     private final boolean mIsRandomRead;
+    private final Random mRandom;
+    private final long mRandomMax;
+    private final long mRandomMin;
```

Comment:
length should be `int`.

Reply:
since these two value will evaluated with long type `mFileSize` later, i think long would be better